indico/htdocs/js/indico/modules/registration/form/tpls/fields/dialogs/size.tpl.html

```diff
@@ -9,3 +9,32 @@
         max="60">
     <i class="info-helper" title="{{ 'Must be between 5 and 60' | i18n }}"></i>
 </div>
+<ng-if="field.inputType == 'text'>
+    <div class="regFormEditLine">
+        <label class="regFormDialogCaption">
+            {{ 'Max length in char' | i18n }}
+        </label>
+        <input type="number" name="maxlength"
+            ng-model="formData.maxLength"
+            >
+        <i class="info-helper" title="{{ 'Maximimum  allowed length' | i18n }}"></i>
+    </div>
+    <div class="regFormEditLine">
+        <label class="regFormDialogCaption">
+            {{ 'Min length in char' | i18n }}
+        </label>
+        <input type="number" name="minlength"
+            ng-model="formData.minLength"
+            >
+        <i class="info-helper" title="{{ 'Minimimum  allowed length' | i18n }}"></i>
+    </div>
+    <div class="regFormEditLine">
+        <label class="regFormDialogCaption">
+            {{ 'Validation pattern' | i18n }}
+        </label>
+        <input type="text" name="validpattern"
+            ng-model="formData.validPattern"
+            >
+        <i class="info-helper" title="{{ 'Validation pattern for input , see https://www.w3schools.com/jsref/jsref_obj_regexp.asp }}' | i18n }}"></i>
```

Comment:
Even though w3schools isn't as awful as they used to be (but the still sell those phony "html certificates"), I don't want a link to them in Indico ;) (also, there shouldn't be a space before the `,`).

Maybe letting users specify regex patterns here is a bit overkill anyway...

Reply:
removed the link. 

Using pattern validation can e.g. be used if you need special input from the registrants (e.g only uppercaser, no spaces, etc.)